modules/highgui/cmake/init.cmake

```diff
@@ -36,10 +36,10 @@ macro(add_backend backend_id cond_var)
   endif()
 endmacro()
 
-add_backend("gtk" WITH_GTK)
```

Comment:
Need to recover GTK plugins support.

Reply:
Yes. I need to push my latest changes. They include working GTK.

Reply:
Sorry. needed quite some to come back to this, but now i pushed the latest changes :)